src/components/StripeBankAccountTokenInputField/StripeBankAccountTokenInputField.js

```diff
@@ -260,11 +260,16 @@ class TokenInputFieldComponent extends Component {
 
     const inputConfiguration = requiredInputs(country);
 
+    // E.g. Japan has 6 fields in the bank account details so we want to
+    // show the inputs in two columns on bigger screens
+    const longForm = inputConfiguration.length >= 6;
```

Comment:
Good thinking 👍 
(Could you create a constant: `const MIN_INPUT_COUNT_FOR_TWO_COLUMNS = 6;` - i.e. all the magic numbers to the beginning of the file.)